internal/database/actions.go

```diff
@@ -82,20 +76,19 @@ func GetMeasurementJSON(sess sqlbuilder.Database, measurementID int64) (map[stri
 		return nil, err
 	}
 	defer file.Close()
-
 	reader := bufio.NewReader(file)
-
 	for {
-		line, err := util.ReadLine(reader)
-		if (err == io.EOF) {
+		line, err := reader.ReadString('\n')
```

Comment:
Real fix: use `ReadString` which [works for long lines](https://twitter.com/_rsc/status/605022288044523520).